.env.example

```
@@ -5,5 +5,5 @@
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
 DATABASE_URL="file:./dev.db"
-# Run 'openssl rand -base64 32' to generate a secret.
-AUTH_SECRET=""
+BETTER_AUTH_SECRET= # Run 'openssl rand -base64 32' to generate a secret.
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)